### DISCUSSION TITLE
Errors during the development

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6677] [Q&A]
Hi everyone, I am new to open-source. Please can someone suggest the solution and what I am doing wrong?
Errors come only after any change to the code is done
![Screenshot 2022-06-05 023943](https://user-images.githubusercontent.com/106697681/172096122-d4974531-2822-41c3-a10d-cd1802f55efb.png)
.

**Comment by Dschoordsch** [comment]
Could you share the changes you've made?
The only related error I've found is https://github.com/facebook/relay/issues/2535#issuecomment-554809124 Did you add the word "graphql" by any chance?

**Reply by 87prashant** [reply]
No, I just changed the CSS width property only
